frontend/views/components/CopyToClipboard.vue

```diff
@@ -0,0 +1,113 @@
+<template lang='pug'>
+.c-wrapper(@click='copyToClipboard')
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='textToCopy'
+  )
+  slot
+  transition(name='fade')
+    i18n.c-tooltip(
+      tag='p'
+      v-if='ephemeral.tooltipActive'
+    ) Copied to clipboard!
+</template>
+
+<script>
+import { activateWebShare } from '../utils/webShare.js'
+
+export default {
+  name: 'copyToClipboard',
+  data () {
+    return {
+      ephemeral: {
+        tooltipActive: false
```

Comment:
Once again it's a nice practice to have bools start with `is` or `has`. in this case `isTooltipActive`. Please review this across the rest of you PR :)

Reply:
Thanks for the info. fixed!